app/src/main/java/com/islaharper/dawnofandroid/ui/theme/Dimens.kt

```diff
@@ -2,16 +2,11 @@ package com.islaharper.dawnofandroid.ui.theme
 
 import androidx.compose.ui.unit.dp
 
+val PADDING_XSMALL = 3.dp
```

Comment:
Are we planning on reusing this somewhere else? Maybe it can be inlined if its only used in one place (and especially since it does't fall within the Material Design guidelines for a grid of 4dp)